Extension/src/LanguageServer/client.ts

```diff
@@ -1086,9 +1086,18 @@ export class DefaultClient implements Client {
                         public provideOnTypeFormattingEdits(document: vscode.TextDocument, position: vscode.Position, ch: string, options: vscode.FormattingOptions, token: vscode.CancellationToken): Promise<vscode.TextEdit[]> {
                             return new Promise<vscode.TextEdit[]>((resolve, reject) => {
                                 this.client.notifyWhenReady(() => {
+                                    const settings: CppSettings = new CppSettings();
+                                    if (settings.formattingEngine !== "vcFormat") {
+                                        // If not using vcFormat, don't process on-type requests
+                                        // for '{' or '\n'
```

Comment:
I was thinking we only did onType for clang-format with `}`.  I believe `;` should be filtered out too.

Reply:
I think you got this backwards.  Looking back at code prior to formatting changes, we only registered for `;`.  Also, the onType test automation currently only checks for `;`.

Reply:
I take it back then. Should have looked at the code first. 😄